.github/workflows/broadcast-frontend-hash_modify-file.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+FILE="$1"
+DFX_VERSION="$2"
+NEW_HASH="$3"
+
```

Comment:
While `awk` is a good choice here, it's easier to write than to read.

It might help to describe at a high level what is going on here.  The pathname tells us roughly "as part of the broadcast-frontend-hash workflow, modify a file."

Example:
```
# In a declaration like the below from https://github.com/dfinity/motoko-playground/blob/main/service/wasm-utils/lib.rs,
# increment the array count and prepend a new entry.
#   const WHITELISTED_WASMS: [&str; 1] = [ 
#      "88d1e5795d29debc1ff56fa0696dcb3adfa67f82fe2739d1aa644263838174b9", // dfx 0.15.0 frontend canister
#.  ];